misc/output_test.py

```diff
@@ -449,6 +449,40 @@ def test_tool_compdb_targets(self) -> None:
             self.assertEqual(expected, actual)
 
 
+    def test_tool_multi_inputs(self) -> None:
+        plan = '''
+rule cat
+  command = cat $in $out
+build out1 : cat in1
+build out2 : cat in1 in2
+build out3 : cat in1 in2 in3
+'''
+        self.assertEqual(run(plan, flags='-t multi-inputs out1'),
+'''out1	in1
+''')
+
+        self.assertEqual(run(plan, flags='-t multi-inputs out1 out2 out3'),
+'''out1	in1
+out2	in1
+out2	in2
+out3	in1
+out3	in2
+out3	in3
+''')
+
+        self.assertEqual(run(plan, flags='-t multi-inputs -F: out1'),
+'''out1:in1
+''')
+
+        self.assertEqual(
+          run(
+            plan,
+            flags='-t multi-inputs -F, --print0 out1 out2'
+          ),
+          f'''out1,in1\0out2,in1\0out2,in2\0'''
```

Comment:
just curious, but why a formatted string here? Does a simple `"out1,in1\0out2,in1\0out2,in2\0"` work?

Reply:
Copy paste error :) same thing is available in test_input_tool ;-)